test/controllers/reports_controller_test.rb

```diff
@@ -17,25 +18,38 @@ class ReportsControllerTest < ActionController::TestCase
   end
 
   test 'employee not found for generate report' do
-    xhr :get, :generate_report, year: @year
+    xhr :get, :generate_report, year: @year, access_token: @user.access_token
     report_response = JSON.parse(response.body)
     assert_equal false, report_response['success']
     assert_equal "Couldn't find User", report_response['error']
   end
 
   test 'comparision to employees sign offs' do
-    xhr :get, :comparision_report, year: @year
+    xhr :get, :comparision_report, year: @year, access_token: @user.access_token
     assert comparision_report = assigns(:employee_comparision)
     report_response = JSON.parse(response.body)
     assert report_response['success']
     sign_offs = @user.sign_offs.includes(:sign_off_type).where("extract(year from date_from) = ?", @year)
-    assert_equal sign_offs.count, comparision_report['john'].values.inject(:+)
+    assert_equal sign_offs.count, comparision_report['johnna'].values.inject(:+)
   end
 
   test 'comparision report not genearte without year' do
-    xhr :get, :comparision_report
+    xhr :get, :comparision_report, access_token: @user.access_token
     report_response = JSON.parse(response.body)
     assert_equal false, report_response['success']
     assert_equal "Couldn't find without year", report_response['error']
   end
+
+  private
+
+  def report_data
```

Comment:
Please give it a meaningful name. Is it a report data or sign off data or creates just sign offs?